### DISCUSSION TITLE
replace all textures with this ASAP it's very urgent

### DISCUSSION THREAD

**Discussion by FuryFire2004** [zer0Kerbal/Dreamer#62] [General]
![image](https://user-images.githubusercontent.com/30536275/159151856-6a484f0d-5569-44f1-82e6-f62bfc642c42.gif)

**Comment by FuryFire2004** [comment]
https://c.tenor.com/r5DCs2wrVtYAAAAd/fortnite.gif

**Comment by zer0Kerbal** [comment]
zer0Kerbal is apolitical - as in doesn't even know what politics are.
Personally I empathize.

**Reply by AtomikkuSan** [reply]
I love the fact that you refer to yourself in the third person!

**Reply by zer0Kerbal** [reply]
> I love the fact that you refer to yourself in the third person!

'got to keep them separated'.. ;P
